frontend/src/features/consultation/ui/chat-panel.module.css

```
@@ -151,33 +153,26 @@
 .systemMessage {
   text-align: center;
   font-size: 0.78rem;
+  letter-spacing: normal;
```

Stylelint (17.7.0)

[error] Duplicate property "letter-spacing" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)